gradle-recommended-product-dependencies/src/main/java/com/palantir/gradle/dist/RecommendedProductDependencies.java

```diff
@@ -29,6 +30,7 @@
 @Value.Style(jdkOnly = true)
 @JsonSerialize(as = ImmutableRecommendedProductDependencies.class)
 @JsonDeserialize(as = ImmutableRecommendedProductDependencies.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I've done these as well as repos use these objects internally with their own object mappers (this will also complicate the rollout of any serialisation change...)